src/libs/actions/SignInRedirect.js

```diff
@@ -16,12 +16,19 @@ Onyx.connect({
     callback: val => currentPreferredLocale = val,
 });
 
+let currentIsOffline;
+Onyx.connect({
+    key: ONYXKEYS.NETWORK,
+    callback: val => currentIsOffline = val.isOffline,
+});
+
 /**
  * @param {String} errorMessage
  */
 function clearStorageAndRedirect(errorMessage) {
     const activeClients = currentActiveClients;
     const preferredLocale = currentPreferredLocale;
+    const isOffline = currentIsOffline;
```

Comment:
Is this const necessary? Can we just use variable declared above in the `Onyx.connect` call?